krabs/krabs/etw.hpp

```diff
@@ -77,6 +77,16 @@ namespace krabs { namespace details {
          */
         EVENT_TRACE_PROPERTIES query();
 
+        /**
+         * <summary>
+         * Configures the ETW trace session settings.
+         * </summary>
+         */
+        void set_trace_information(
+            TRACE_INFO_CLASS informationClass,
```

Comment:
let's use snake_case here to match the style of the surrounding code: `information_class`, `trace_information`, `information_length`

Reply:
Done.